tools/src/h5repack/h5repack.c

```diff
@@ -478,8 +486,15 @@ copy_attr(hid_t loc_in, hid_t loc_out, named_dt_t **named_dt_head_p, trav_table_
             if (buf == NULL) {
                 H5TOOLS_GOTO_ERROR((-1), "HDmalloc failed");
             } /* end if */
+            if (options->verbose == 2)
+                HDgettimeofday(&timer_start, NULL);
```

Comment:
gettimeofday() should really not be used for timing. It's better to use the code in H5timer.c instead.

Reply:
POSIX.1-2008 marks gettimeofday() as obsolete, btw.